appscale/tools/appscale_tools.py

```diff
@@ -630,6 +631,65 @@ def remove_app(cls, options):
 
     AppScaleLogger.success('Done shutting down {}.'.format(options.project_id))
 
+  @classmethod
+  def start_service(cls, options):
+    """Instructs AppScale to start the named service.
+
+    This is applicable for services using manual scaling.
+
+    Args:
+      options: A Namespace that has fields for each parameter that can be
+        passed in via the command-line interface.
+    Raises:
+      AppScaleException: If the named service isn't running in this
+        AppScale cloud, or if start is not valid for the service.
+    """
+    login_host = LocalState.get_login_host(options.keyname)
+    secret = LocalState.get_secret_key(options.keyname)
+    admin_client = AdminClient(login_host, secret)
+
+    version = Version(None, None)
```

Comment:
Should this just be `Version(None)`? It looks like it just takes the runtime parameter.

Reply:
I think `Version(None, None)` is correct for `master`:

```
class Version(object):
  """ Represents an Admin API Version resource. """
  def __init__(self, runtime, config_type):
```

https://github.com/AppScale/appscale-tools/blob/master/appscale/tools/admin_api/version.py#L22

Reply:
You're right. I was looking at an older commit.